'Solution'

Irina17191 opened the pull request.

Oleg-s-s commented on app/main.py:
```diff
@@ -1 +1,50 @@
+
+
+class FullStackDeveloper(BackendDeveloper, FrontendDeveloper):
+    def __init__(self, name: str) -> None:
+        BackendDeveloper.__init__(self, name)
+        FrontendDeveloper.__init__(self, name)
+        self.skills = (BackendDeveloper(name).skills
+                       + FrontendDeveloper(name).skills)
```
You sure you need this?)

Oleg-s-s commented on app/main.py:
```diff
@@ -1 +1,49 @@
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = ["JavaScript", "HTML", "CSS"]
```
You should extend skills, not recreate

Oleg-s-s commented on app/main.py:
```diff
@@ -1 +1,49 @@
+class BackendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = ["Python", "SQL", "Django"]
```
same here

Oleg-s-s commented on app/main.py:
```diff
@@ -1 +1,49 @@
+class AndroidDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = ["Java", "Android studio"]
```
same here

Oleg-s-s commented on app/main.py:
```diff
@@ -1 +1,49 @@
+
+
+class FullStackDeveloper(BackendDeveloper, FrontendDeveloper):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = (BackendDeveloper(name).skills
+                       + FrontendDeveloper(name).skills)
```
If you will do that properly higher, maybe you will manage to 'shorten' this much, much more)

Irina17191 commented:
> > I don't see your fixes according to previous comments. Please read all the comments that mentors posted earlier and push fixed code
> 
> But I fixed everything from the previous comments and even visited qna today to figure out with this task. (I added extend methods where it's needed and changed class FullStackDeveloper in order to write it shorter)

Probably I didn't save my changes correctly previous time, I'm sorry)

Irina17191 commented:
> > Probably I didn't save my changes correctly previous time, I'm sorry)
> 
> yeah, you didn't push your changes, that's why I was not able to see them :)

I'm sorry)  and thank you!)
